yelp/google_cloud_v2/main.py

```diff
@@ -0,0 +1,82 @@
+import json
+import requests
+
+
+def make_request(url, api_key):
+    headers = {"Authorization": "Bearer {0}".format(api_key)}
+    r = requests.get(url, headers=headers)
+    return json.loads(r.text)
+
+
+def assemble_response_json(insert, state):
+    response_dict = {
+        "state": state,
+        "schema": {
+            "businesses": {
+                "primary_key": [
+                    "id"
+                ]
+            },
+            "business_category": {
+            }
+        },
+        "insert": insert,
+        "hasMore": False
+    }
+    return json.dumps(response_dict)
+
+
+def handler(request):
+
+    root_url = "https://api.yelp.com/v3/businesses/search"
+    location = "San+Francisco"
+    type = "makerspaces"
+
+    # GET THE CREDENTIALS FROM THE TRIGGERING EVENT
+    request_json = request.get_json()
+
+    api_key = request_json['secrets']['api_key']
+
+    url = "{0}?location={1}&categories={2}".format(root_url, location, type)
+
+    # REQUEST DATA FROM API
+    response_text = make_request(url, api_key)
+
+    # ITERATE THROUGH LIST OF BUSINESSES
+    results = response_text['businesses']
+
+    business_table = []
+    bus_cat_join_table = []
+
+    for result in results:
+        result.pop("transactions", None)
+
+        categories = result.pop("categories", None)
```

Comment:
Let's move this line down towards where you iterate over `categories` so it reads a bit clearer.

Reply:
Thanks!